source/getting-started/tutorials/sensors/index.md

```diff
@@ -1,52 +1,274 @@
-# Tutorial: Sensor Configuration, Feedback, & Profiling
+# Tutorial: Timing & Sensors
 
-- **Goal:** Learn how to use the AMDC Timing Manager to get feedback from sensors
-- **Complexity:** 4 / 5
-- **Estimated Time:** TODO
+- **Goal:** Learn how to use the AMDC Timing Manager to configure, profile, and get feedback from sensors
+- **Complexity:** 3 / 5
+- **Estimated Time:** 40 min
 
 
 ## Tutorial Requirements
 
 1. Working AMDC Hardware
 2. Completion of the [Hardware Commands](/getting-started/tutorials/hw-commands/index.md) tutorial
 3. Completion of the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial
+4. Completion of the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial
+
+This tutorial expands on the code created in the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial, and uses commands created in the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial. Both must be completed before this tutorial.
 
 ## Introduction 
 
-## Step 1:
+In motor control applications, it is important for the timing of both sensor data acquisition and control task execution to remain consistent. In its default configuration, the AMDC may allow some jitter (timing incosistency) between sensor measurement time and control loop timing. This tutorial shows how the AMDC's "Timing Manager" peripheral can be configured to eliminate this jitter.
+
+This tutorial provides code that allows users to experiment with the configuration of the AMDC Timing Manager. The effects of this experimentation will be observed through the lens of the previous [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial.
+
+## Scheduling and Synchronizing:
+
+The AMDC synchronizes sensor data collection and task execution to the PWM carrier wave. Every X PWM periods (where X is set by the function `timing_manager_set_ratio()`), the AMDC will collect data from sensors. By default, the Timing Manager is set to Legacy Mode, in which the AMDC will run control tasks concurrently with sensor data collection. Alternatively, when configured in Post-Sensor Mode, the Timing Manager guarantees that in each control cycle, the AMDC will wait to execute control tasks until the sensor data collection is complete.s
+
+There are multiple factors that affect when and how fast control tasks run.
+ - User set `TASK_UPDATES_PER_SEC`
+ - PWM Frequency
+ - User Event Ratio
+ - Sensor collection time (Post-Sensor Mode)
+ - Control task time (how long it takes for the control task to run)
```

Comment:
I find this list / section confusing. As a reader, I am not really sure what you want me to take away from it.  We need some revision to make this more user friendly. 

- `TASK_UPDATES_PER_SEC` doesn't exist. I think you mean `TASK_CONTROLLER_UPDATES_PER_SEC` ?
- I am not sure the timing manager diagram below is really helping anything here. Maybe not needed?

Reply:
`TASK_UPDATES_PER_SEC` was intended to be a generalised version of the macro for any task. In our specific case, the macro is called `TASK_CONTROLLER_UPDATES_PER_SEC` because our app is called controller. In the blink app, it is called `TASK_BLINK_UPDATES_PER_SEC`.